backend/doc/user_custom_properties.md

```diff
@@ -0,0 +1,98 @@
+# User Custom Properties
+User custom properties is a feature allowing to add instance-specific properties
+for all user in the instance.
+
+Idea is that for each user is associated some properties
```

Comment:
```suggestion
Idea is: for each user are associated some properties
```

Reply:
```suggestion
Idea  is: properties are associated to each user
```

Reply:
`The structure of those user properties should be defined in a json schema file.
It is possible to fine-tune the display and edition behavior of those properties by the Tracim frontend in a separate file (named UI schema). Its syntax can be found [here](https://react-jsonschema-form.readthedocs.io/en/latest/api-reference/uiSchema/)`

Reply:
I made a suggestion based on @grignards proposal.